### ISSUE TITLE
Karma and Todos for Roded Zats

### ISSUE THREAD

**Issue by zvili** [TechnionYP5777/Bugquery#10]
_From @rodedzats on October 30, 2016 20:53_

- [x]  Use a picture, photo or distinct avatar. Don''t use an automatic icon:
- [x]  Make sure you present yourself by a readable **nick**
- [x]  Describe yourself in the bio page; FYI, here is my description:
- [x]  State that you are (not) in search for a job, i.e., mark, or unmark this box:

_Copied from original issue: SpartanRefactoring/Spartanizer#656_

**Comment by zvili** [comment]
_From @rodedzats on October 30, 2016 20:53_

@YossiGil: Done.

**Comment by zvili** [comment]
_From @tomerdragucki on October 30, 2016 20:58_

Hello friend, I liked your bio!

**Comment by zvili** [comment]
_From @Shahar-Y on October 31, 2016 15:48_

Awesome karma page! LOVED the Hello Kitty theme!
#665

**Comment by zvili** [comment]
_From @AdiOmari on November 1, 2016 7:58_

@rodedzats  Add teaching and quality assurance labels. Other than that, looks good 👍

**Comment by zvili** [comment]
_From @yossigil on November 26, 2016 11:47_

@zvili : please move to student's current repo

**Comment by yossigil** [comment]
Can you summarize here all issues you dealt with towards Sprint 1?

@AdiOmari

**Comment by rodedzats** [comment]
@yossigil 
I've been working with @yonzarecki on #8 and on #14
